README.md

```diff
@@ -21,6 +21,14 @@ This code was defined in 1998 as one of the traditional IETF April Fools' jokes,
 - Download a ready to use image from the [release page](http://github.com/zeroflag/Teapot/releases/latest)
 - Explore the [documentation](docs/).
 
+```
+Metacello new
```

Comment:
Thanks for the PR @Ducasse. We already have installation instructions under `docs/Installation.md` but that might be more difficult to find so I think it makes sense to include it here too.